### DISCUSSION TITLE
Quectel L76K | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1167] [Q&A]
# Quectel L76K | Seeed Studio Wiki

Get Started with L76K GNSS Module for XIAO

https://wiki.seeedstudio.com/get_start_l76k_gnss/

**Comment by GustavoMurta13** [comment]
The information appears to come incomplete. The example displays the information and the location comes INVALID but the date/time comes valid and constantly updating correctly.
My setup is the same as estipulated in this tutorial with the XIAO ESP32C3.

**Reply by Love4yzp** [reply]
Hey, are u working with it? any log to share?

**Comment by agalliani** [comment]
If the pin Sleep is set to 0, the green led is switched on until the pin is set to Normal (1). Is there a way to shut it down? The provided example does not work.

**Reply by Love4yzp** [reply]
They are different LEDs.

**Comment by dpcrd** [comment]
What is the purpose of the 5V pin? According to the schematic, it's connected to nothing on the board.

**Reply by nfs0619** [reply]
Thanks for your inquiry! The 5V pin on the module corresponds to the 5V pin on the XIAO, as the module is designed to be used in conjunction with the XIAO.

**Reply by krazykonr** [reply]
Wish I had seen this earlier, worked it out the hard way with the module in hand... :P

**Comment by palmacas** [comment]
Is there any chance to get the 3D model of the module and the antenna dimensions?

**Reply by ReKST-117** [reply]
I am assuming that is not available if the file is not listed in the 'Resources' chapter under Wiki page.

**Comment by krazykonr** [comment]
As an FYI for anyone using this module not with a XIAO board - 3V3 input required. 

Seems that the 5V pin is completely unused by the board.

**Comment by janlhgm** [comment]
## Meshtastic GPS setup with XIAO nRF52840 + Wio-SX1262
I’m trying to get the L76K GPSworking with aSeeed XIAO nRF52840 + Wio-SX1262 Kit running Meshtastic.

The wiring should be correct, and I’m using:
D6 → GPS TX
*D7 → GPS RX
In Meshtastic I enabled GPS/Position, but I’m not getting a GPS fix or usable GPS data.

Has anyone successfully configured an **XIAO nRF52840 + Wio-SX1262 + L76K** with the current Meshtastic firmware?

What are the correct **GPS pins and Meshtastic settings** for this combination?

**Reply by Yvesgoat** [reply]
The wiring you described is not quite correct for this kit.

For the XIAO nRF52840 + Wio-SX1262 + L76K combination, the intended hardware connection is:

XIAO nRF52840 → Wio-SX1262 → L76K

In other words, you should first connect the Wio-SX1262 to the XIAO nRF52840, and then connect the L76K GPS module to the GPS interface on the Wio-SX1262 board.

So the L76K should not be wired directly to D6/D7 on the XIAO in this setup.

Please try connecting the hardware in this order first, then enable GPS / Position in Meshtastic and test again outdoors with a clear view of the sky.
